src/views/study/dataset/metadata/about/DatasetMetadataOther.vue

```diff
@@ -159,7 +143,7 @@ const saveMetadata = (e: MouseEvent) => {
         of `Diabetes` yields `Diabetes dataset`.
       </p>
 
-      <n-form-item label="Name" path="managing_organization_name">
+      <n-form-item label="Name" path="resource_type" show-require-mark>
```

Comment:
is `show-require-mark` needed here? I belive the path should add that by default since it is required in the rules

Reply:
Ahh that is true. Removed